### PR TITLE
[release/3.1] Fix component government bugs

### PR DESCRIPTION
**Pull request by wtgodbe**
Fix (at least) most of the Medium+ release/3.1 CompGov bugs

### CODE REVIEW

File: src/Components/Web.JS/package.json [hunk 1]
```diff
@@ -29,13 +29,16 @@
     "ts-loader": "^4.4.1",
     "typescript": "^3.8.3",
     "webpack": "^4.42.1",
-    "webpack-cli": "^3.3.11"
+    "webpack-cli": "^3.3.11",
+    "yarn-audit-fix": "^9.2.2"
```
Comment (dougbu) [inline src/Components/Web.JS/package.json:33]: Do not add this dependency. `yarn-audit-fix` should be in your global cache but not listed as a dev dependency in this repo.

**Review by dougbu — COMMENTED** [review]
I can't sign off as long as `yarn-audit-fix` is listed as a dev dependency in any package.json file

**Comment by wtgodbe** [comment]
> I can't sign off as long as yarn-audit-fix is listed as a dev dependency in any package.json file

Should be fixed

**Review by dougbu — APPROVED** [review]
If you can get this building&hellip;

**Comment by wtgodbe** [comment]
The build failures are very weird, they started happening after just updating some yarn.lock files, and they're right at the beginning of the build, before we'd be trying to do anything with the .js/.ts stuff:

> C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin\amd64\msbuild.exe /nologo /bl:D:\a\_work\1\s\artifacts\log\Release\ToolsetRestore.binlog /clp:Summary /clp:ErrorsOnly;NoSummary /m /nr:False /p:ContinuousIntegrationBuild=True /p:TreatWarningsAsErrors=false /p:__ToolsetLocationOutputFile=D:\a\_work\1\s\artifacts\toolset\1.0.0-beta.22212.4.txt /t:__WriteToolsetLocation /v:minimal D:\a\_work\1\s\artifacts\toolset\restore.proj
D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\1.0.0-beta.22212.4\tools\Tools.proj : error MSB4236: The SDK 'Microsoft.NET.Sdk' specified could not be found.
##[error].packages\microsoft.dotnet.arcade.sdk\1.0.0-beta.22212.4\tools\Tools.proj(0,0): error MSB4236: The SDK 'Microsoft.NET.Sdk' specified could not be found.
##[error]Build failed.

Gonna try the 1es images

**Comment by wtgodbe** [comment]
Kestrel tests like `TlsHandshakeRejectsTlsLessThan12` are now failing consistently with:

> System.ComponentModel.Win32Exception : The client and server cannot communicate, because they do not possess a common algorithm.

This rings a bell, but I can't find old info about it - @halter73 @BrennanConroy @Tratcher does this look like a machine issue? It looks like these tests were deleted some time in the 5.0 time frame

**Comment by wtgodbe** [comment]
@Tratcher @BrennanConroy @halter73 @JamesNK any ideas on the above?

**Pull request merged**